crates/matrix-sdk-crypto/Cargo.toml

```diff
@@ -59,7 +59,7 @@ features = ["js"]
 
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies.ruma]
 version = "0.6.1"
-features = ["client-api-c", "rand", "unstable-msc2676", "unstable-msc2677"]
+features = ["client-api-c", "rand", "unstable-msc2676", "unstable-msc2677", "signatures"]
```

Comment:
```suggestion
features = ["client-api-c", "rand", "signatures", "unstable-msc2676", "unstable-msc2677"]
```

Reply:
This one as well https://github.com/matrix-org/matrix-rust-sdk/pull/716/commits/1223191f17eeb7559b1701b1445033a91053af39.